container-images/candlepin/start

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z $CANDLEPIN_TRUSTSTORE_PASSWORD ]] ; then
+	CANDLEPIN_KEYSTORE_PASSWORD=$CANDLEPIN_TRUSTSTORE_PASSWORD
+	export CANDLEPIN_KEYSTORE_PASSWORD
+fi
+
+for template in /etc/candlepin/*.tpl ; do
+	destination=$(basename "$template" .tpl)
+	if [[ ! -f "$destination" ]] ; then
```

Comment:
This was intended to allow overrides but now I realize it may be created during container creation